generators/client/templates/vue/src/main/webapp/app/components/admin/audits/audits.component.ts.ejs

```diff
@@ -17,6 +15,7 @@ export default class <%=jhiPrefixCapitalized%>Audits extends mixins(AuditsServic
   public reverse: boolean;
   public toDate: any;
   public totalItems: number;
+  @Inject('auditsService') private auditsService: () => AuditsService;
```

Comment:
Could you move injected field to the top of the class in order to be consistent with other injected files in other components ?

Reply:
Oops that's my fault, I'm used to have `private` fields declared after the `public` ones.

Reply:
We discuss about that with @hdurix, we prefer put private attributes after public.

Reply:
Ok we'll clear this later on another PR :)